third_party/move/move-compiler-v2/src/bytecode_generator.rs

```diff
@@ -1047,7 +1047,7 @@ impl<'env> Generator<'env> {
             _ => self.error(
                 id,
                 "assignment to tuple must be tuple itself or a function call",
-            ),
+            ), // TODO: allow assignment to a tuple from move_to, abort, etc.
```

Comment:
I'll create a separate issue for this, but when I tried to add the visibility checks on all type check only tests, it exposed a bug where we don't run stackless bytecode gen (sbcodegen) on those programs, and sbcodegen itself could report errors.

Turns out many of those errors are not valid. Eg, `let () = move_to(...)` should be valid, but is not because of this code flagged.

Reply:
I was thinking the other way round would be better: link this code in the issue (which I have yet to create, along with a bunch of others), and then we can do all the issue management we want.

Reply:
Yes, those places were not following this practice, but I agree with Brian that moving forward we should do this.